### PR TITLE
Dive_Deep_Into_CircleStarks

### PR DESCRIPTION
**Pull request by YangXiaoXue205**

### CODE REVIEW

File: Dive_Deep_Into_CircleStarks/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# {Project Name}
```
Comment (rutefig) [inline Dive_Deep_Into_CircleStarks/README.md]: can you add the name of the project here pls? (pls remove the `{` when you replace the requested data

File: Dive_Deep_Into_CircleStarks/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+We are SECBIT Labs. We aim to build high-confidence and trustworthy protocol, application, and facilities on the blockchain. We focus on the research of formal verification, practices of zero-knowledge proofs and blockchain security. We have explored the full formal verification for token smart contracts and decentralized exchanges for the first time. We also have designed and implemented the zkPoD, a practical decentralized protocol for data exchange. We reported many buggy smart contracts on Ethereum, including well-known Fomo3D bugs and attack vectors.
+Email: <hi@secbit.io>
+GitHub: <https://github.com/sec-bit>
+Twitter: <https://x.com/SECBIT_IO>
+Medium: <https://medium.com/@SECBIT>
+Zhihu: <https://www.zhihu.com/org/secbit/posts>
+Wechat/Weixin Articles Archive: <https://mp.weixin.secbit.io/>
+**Project Members**
+
+- Name: {YangXiaoXue}
```
Comment (rutefig) [inline Dive_Deep_Into_CircleStarks/README.md]: Pls remove the `{` `}` after the data being replaced

**Review by rutefig — CHANGES_REQUESTED** [review]
Other that this small things everything is good 🚀

**Comment by rutefig** [comment]
@YangXiaoXue205 can you check the comments pls?

**Comment by YangXiaoXue205** [comment]
Sorry, I just saw the comment, I just commited it

**Review by rutefig — APPROVED** [review]

**Pull request merged**